pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.md.erb

```diff
@@ -167,14 +186,275 @@ The Code.org Professional Learning Program is open to educators who are interest
 * For CS Principles, teach students between 9th and 12th grade
 * Support the recruitment and enrollment of a diverse group of students in the course, representative of the school's student population
 
+[/optimizely-original]
+
+[optimizely-variant-a]
```

Comment:
Contents of variant A were copied from [middle-high-2020.md.erb](https://github.com/code-dot-org/code-dot-org/blob/97ab67caf032637d664896ff013da4fbb710a026/pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high-2020.md.erb).  Engineering changes to this variant were [reviewed here](https://github.com/code-dot-org/code-dot-org/pull/31930).